frontend/src/components/taskSelection/footer.js

```diff
@@ -110,6 +110,14 @@ const TaskSelectionFooter = ({ defaultUserEditor, project, tasks, taskAction, se
           }
         })
         .catch((e) => lockFailed(windowObjectReference, e.message));
+
+      if (project.imageCaptureMode) {
+        if (navigator.userAgent.includes('Android')) {
+          navigate('mapillary://mapillary/explore');
+        } else if (navigator.userAgent.includes('like Mac OS X')) {
```

Comment:
I think that we want to check that it's an iPhone or iPad, so let's try this instead:
```suggestion
        } else if (navigator.userAgent.includes('iPad|iPhone|iPod')) {
```